master/internal/experiment/bulk_action.go

```diff
@@ -369,7 +402,9 @@ func DeleteExperiments(ctx context.Context, system *actor.System,
 	if filters == nil {
 		query = query.Where("e.id IN (?)", bun.In(experimentIds))
 	} else {
-		query = queryBulkExperiments(query, filters)
+		query = queryBulkExperiments(query, filters).
+			Where("versions = 0").
```

Comment:
i think we don't want this `Where`, because we check for it later and want to explicitly call this out to users? unless i am miss understanding this.. massive query.